yew-functional/src/hooks/use_effect.rs

```diff
@@ -0,0 +1,56 @@
+use crate::{use_hook, Hook, HookUpdater};
+
+pub fn use_effect<F, Destructor>(callback: F)
```

Comment:
Can't we just have this call `use_effect_with_deps(callback, ())`?

Reply:
Hmmm... not quite. I tried but the tests didn't pass. When passed empty dependencies, that correlates to a one-time call on component rendering. In fact, this is how use_effect works in react - an empty dep array is an "on-mount" call.